docs/project/code-migrate-1.0-sdk.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Code Migration Guide
```

Comment:
```suggestion
title: Migrating to 1.0
```

Reply:
@ozkatz can we say "API" and/or "SDK" in the title somewhere? I want to make it clear to non-programmers that they don't need to read this. It will be more confusing to someone who just uses Spark/lakeFS.
```suggestion
title: Migrating to SDK version 1.0
```

Reply:
think we should keep it short as it will be on the left side menu